src/FileServiceManager/Models/FileMetadata.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace FileService.Models
+{
+    public class FileMetadata
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Extension { get; set; }
+        public string UploadedBy { get; set; }
+        public DateTime? CreatedOn { get; set; }
+        public string FilePath { get; set; }
+        public byte[] Content { get; set; }
```

Comment:
Metadata is about everything , but not the content itself

Reply:
There was misunderstanding during our previous conversations. After last call, it becomes more clear. I remove content.